cyences_app_for_splunk/default/data/ui/views/cs_kaspersky_reports.xml

```diff
@@ -0,0 +1,157 @@
+<form theme="dark" version="1.1">
+  <label>Kaspersky</label>
+  <fieldset submitButton="false">
+    <input type="time" token="tkn_timerange">
+      <label></label>
+      <default>
+        <earliest>-24h@h</earliest>
+        <latest>now</latest>
+      </default>
+    </input>
+  </fieldset>
+  <row>
```

Comment:
Add panel with the status and heart-beat data. (Health Check panel) - use the Device Inventory lookup if possible.